vita3k/gui/src/game_selector.cpp

```diff
@@ -167,6 +174,13 @@ void draw_game_selector(GuiState &gui, HostState &host) {
             bool selected[4] = { false, false, false, false };
             if (!gui.game_search_bar.PassFilter(game.title.c_str()) && !gui.game_search_bar.PassFilter(game.title_id.c_str()))
                 continue;
+            if (!fs::exists(fs::path(host.pref_path) / "ux0/app" / game.title_id)) {
+                gui.title_id = game.title_id;
+                LOG_ERROR("Game not found: {} [{}], deleting the entry for it.", game.title_id, game.title);
+                delete_game(gui, host);
+                if (gui.delete_game_popup)
```

Comment:
no

Reply:
Why no ? I have set this for good reason

Reply:
for this, i have set this code, for prevent crash,n if user delete folder, but have already open delete game context ^^